game/scripts/vscripts/Sections/Chapter1/SectionLeveling.ts

```diff
@@ -64,7 +95,7 @@ const start = (complete: () => void) => {
                     tg.loop(ctx => unitIsValidAndAlive(ctx[CustomNpcKeys.PurgePugna]), tg.audioDialog(LocalizationKey.Script_1_Leveling_4_2, LocalizationKey.Script_1_Leveling_4_2, ctx => ctx[CustomNpcKeys.PurgePugna])),
                     // Stop long dialog when we get attacked but not stunned. Let purge speak for at least a few seconds too.
                     tg.seq([
-                        tg.wait(2),
+                        tg.wait(0.1),
```

Comment:
2s was intentional here, so he starts talking for a bit instead of spamming "please dont interrupt me"

Reply:
This was actually an issue when you were immediately auto attacking him when he spawned, but that was resolved. Now he says "please don't interrupt me" once every 2 hits, which is about the time it takes for him to say that line.

Works fine on my end.

Reply:
REVERTED